k8s/helm/templates/frontend-server-service.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: frontend-server
+  namespace: chroma
+spec:
+  ports:
+    - name: frontend-server
+      port: 8000
+      targetPort: 8000
+  selector:
+    app: frontend-server
+  type: LoadBalancer
+
+---
+
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
wdyt about changing this to `StatefulSet`? Here are some of the benefits over Deployment:

 - Stable storage
 - Stable network IDs
 - Ordered, graceful deployment and scaling.

In practice, I think the storage part is the most important here as we need ordered instances that when restarted will always get assigned the same storage (PV/PVC)